apps/clearance_ui/src/@types/table-core.d.ts

```diff
@@ -0,0 +1,27 @@
+// SPDX-FileCopyrightText: 2024 Double Open Oy
+//
+// SPDX-License-Identifier: MIT
+
+import { RowData } from "@tanstack/react-table";
+
+declare module "@tanstack/table-core" {
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    interface TableMeta<TData extends RowData> {
+        editedRows: { [key: number]: boolean };
+        setEditedRows: React.Dispatch<
+            React.SetStateAction<{ [key: number]: boolean }>
+        >;
```

Comment:
Does `editedRows` mean the rows that are selected for editing? If so, `selectedRows` or something similar could be more descriptive.

Reply:
Yes, changed the name for this now